README.md

```diff
@@ -1,4 +1,20 @@
 # GitHub Action - Releases API
+
+**Please note:** This repository is currently unmaintained by a team of developers at GitHub. The 
+repository is here and you can use it as an example, or in Actions. However please be aware that 
+we are not going to be updating issues or pull requests on this repository.
+
+**Maintained forks:**
+* ...
```

Comment:
Please suggest some maintained community forks, we'll add them in here.

Reply:
@hross I guess you are the maintainer here. Please can you add some other maintainers. Maybe the last 5 guys who contribute already ?

Reply:
Sadly this is the current/future state of most github repos: The maintainer looses interest and the project dies slowly.
There is no relay race culture to prevent from dead project

Reply:
@hannesa2, see https://github.com/actions/upload-release-asset/graphs/contributors.